pkg/controller/ingress-shim/sync.go

```diff
@@ -153,6 +153,11 @@ func (c *controller) buildCertificates(ctx context.Context, ing *networkingv1bet
 					Kind:  issuerKind,
 					Group: issuerGroup,
 				},
+				Usages: []cmapi.KeyUsage{
+					cmapi.UsageDigitalSignature,
+					cmapi.UsageKeyEncipherment,
+					cmapi.UsageServerAuth, // default for web facing certificates as per https://support.apple.com/en-us/HT210176
+				},
```

Comment:
Will this cause all certificates to be re-issued?

Reply:
not entirely sure as we kept key usages quite special as this is the most often not followed by the CA, we didn't even pass it along in the CSR until 1.0

Reply:
Can we add some test cases to the trigger controller for this? Create a CertificateRequest using the old options and then another with the new and see if it triggers?

Reply:
It'd be particularly bad for users if it _does_ trigger a re issuance. It'd be awesome if we had some upgrade testing somewhere for this sort of thing in a more end to end way...

Reply:
Great idea! will do that
Yeah upgrade testing would be good.... it has been on our backlog for a while now

Reply:
Added an e2e test